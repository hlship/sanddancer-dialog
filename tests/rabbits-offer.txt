(setup lighting)
take jade
brood stories
teleport burrow
smell
smell fur
x hare
go insane
ask who
real
ask much
no
trade jade for courage
trade jade for strength
trade lighter for strength
trade lighter for rabbit
trade stories for strength
i
x roots
climb them
look up
