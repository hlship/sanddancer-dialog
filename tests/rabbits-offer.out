

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> (setup lighting)
Query succeeded: (setup lighting)
> take jade
You take the piece of jade.

You shiver in the cold.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> brood stories
There are dark spirits who roam the earth, little Knock. Grandma used to say
that, holding you tight and stroking your hair. There are dark spirits who roam
the earth, but you're not alone. Oh, no. I'm here. (She'd kiss your head and
you'd squeeze her back.) But others are watching out for you too. You have three
animal guardians, hmm? Spirits who are always watching over you. Oh, you can't
always trust them to know what's best. Remember that, Knock. But when you need
help, they'll come, and protect you from the worser things in the world.

Mom would yell at Grandma a lot for filling your head with that new-age
bullshit. Grandma grew up white and midwestern and Baptist, but had started
wearing things with feathers and playing the pan flute by the time your were
born. She seemed to really like having a son-in-law who was Native American or
American Indian or Indigenous People or whatever she'd decided the term was that
week, and she was pretty pissed when Mom left him. Anyway. Her stories were
mostly BS, you guess, but some of them stuck with you. When it's dark you still
wonder if your spirit animals are out there, somewhere, and what the hell is
taking them so long to find you.

> teleport burrow

Burrow
Roots push through the earthen roof, casting weird shadows in the beam of your
flashlight. It's round and small and underground in here and filled with hot
sweat and animal stench. But mostly filled with something big and alive.

It's huge, covered in sweaty fur stained with mud, and a huge eye stares out you
above long black whiskers. You can feel its breath on your face and its
heartbeat thudding through the floor and you cringe back in terror against the
wall. It'sâ€¦ well.

It's a rabbit.

> x rabbit
The closer you look the weirder it gets, like it doesn't start or end in any one
place or something all new-agey like that.

The rabbit breaths and stares. You wonder if it's going to start talking like in
some crap disney movie and then jesus christ it does...

"So, what'd you expect from an animal guardian?" it says, but like not with its
lips, somewhere in your head instead. "Were you thinking big, nasty, sharp,
pointy teeth? Waistcoat and pocket watch? Nah. Tricks are for kids. What's up
Knock?" Its whiskers twitch.

You could ask who the rabbit is or just, you know, go insane.

> go insane
I know, right? Looks like it's too late though.

You could ask who the rabbit is.

> ask who
"Who, uh, what are you?" you ask.

"Oh come on, pal, you know me," he says with this kind of stupid cheerful tone,
and suddenly he looks more like Buster from Arthur, scrawny and poorly animated.
"I've been keeping an eye on you from the TV all these years. Your grandma asked
me to, remember?" He grins, buck teeth protruding.

You could ask about grandma's stories or tell the rabbit he's not real.

> ask real
(I only understood you as far as wanting to ask someone something.)

> tell real
"But, you're not, like, real."

"Oh, don't say that!" the rabbit says, animated eyes going round like saucers.
"Then it'd be just you here all alone, and wouldn't that be boring. It's much
more interesting if I'm really around."

A moment passes.

"Hey, yo, you been hella good about keepin' off those cancer sticks," Buster
says, except he's not Buster any more, more like Eminem. "Don't think I ain't
watchin' ya." (Oh hey, Rabbit Smith, you get it.)

You could ask how he knows so much about everything.

> ask much
"I don't understand," you ask "how you know so much about me. Grandma.
Everything."

"Hey, back off man," he says, "it's my ***n' job." And there's actually like a
bleep, too, and now that you think of it you only ever saw 8 Mile on TV.

A moment passes.

"Anyway, look, man, we both know you're in trouble. I just gotta ask you one
question." And he changes again into like a giant silhouette, and it's the
Playboy Bunny, ears and bow tie and all. "Do you love her?"

You could tell him that you don't know, answer yes, or answer no.

> no
"I tell her I love her," you begin, "because I know she needs me to say it. But
really, deep down, I know it shouldn't be because of her. It should be because
of me, and that's just not there. Not yet. Maybe not ever."

Something about the bunny's bow tie makes you want to brood about her graduation
night.

"Then what are you still doing with her?" the bunny says. "Get out while the
gettings good. You think either of you will be happier if you put that off?
There's fish in other oceans kid. You've got your whole life to swim." It winks.

A moment passes.

"Look," he says, and now he's big and fluffy and smells like springtime and
peeps. "I can help you make your decisions. I'll trade you for one of these
talents." He holds out an easter basket filled with plastic green easter grass;
in it are two glowing orbs.

"Strength," hey says, pointing to the first one, "and courage. Pick whichever
one you think will be most useful, but you have to give me something back.
Something real, something I can work with."

You could ask about strength, ask about courage, ask about trading, or tell him
you don't want to trade.

> trade stories for strength
The Rabbit nods. "Yes," he says, "a fair trade." And something happens inside
you as he says it. The memory of your grandma's stories shifts and wriggles and
fades away and it's still there, but now it's shifted, become something else.
And, yeah, it kinda does feel like you could call it strength. Cool.

"The others are coming," he says, "they'll be here soon," and now he looks like
Frank from Donnie Darko and his face is behind that freaky metal mask and his
voice is distorted and makes your skin crawl.

"Watch out for them," he says. "We all want to help, but... the others are
wilier than me. Look up."

"Look up, Naki," he says again.

> i
You have strength, a piece of jade, a flashlight, your lighter, your wallet, and
your emotional baggage. You're wearing your denim jacket.

In your emotional baggage is her graduation night.

> look up
Stupidly, you tilt your head back and stare up at the dirt ceiling, and all you
can see is blackness.

No wait... there's patterns, slowly shifting, barely visible in the darkness,
and miles away. Like clouds. And then a drop of water falls on your face, and
then another, and you realize suddenly you're outside, and it's starting to
rain...

Backtracking
It's quieter and darker over here, the black desert night bleeding in. Your tire
track are hardly visible in the desert sand, and you can barely make out the way
north back to your truck.

Rain starts to come down, pattering on the desert sand. Soon it's gonna be a
muddy deathtrap out here.

> 
